docs/java-rest/high-level/rollup/put_job.asciidoc

```diff
@@ -119,6 +119,68 @@ include-tagged::{doc-tests}/RollupDocumentationIT.java[x-pack-rollup-put-rollup-
 <2> Adds the metrics to compute on the `temperature` field
 <3> Adds the metrics to compute on the `voltage` field
 
+`RollupJobConfig` also sets default metrics for `DateHistogram` and `Histogram` groupings.
+If the user does not provide a `MetricConfig` referencing the field in a `DateHistogram` or
```

Comment:
Could we reword this slightly so it is shorter sentences? Maybe something like:

>By default, `min`/`max`/`value_count` metrics will be added to the `DateHistogram` and `Histogram`.  This behavior can be overridden by explicitly configuring metrics for the `DateHistogram` and `Histogram` groups.  

Or something.  The "If the user..." sentence just ran a bit long and was hard to parse.